components/SignUpForm.tsx

```diff
@@ -108,6 +115,28 @@ const SignUpForm: React.FC = () => {
               <p>Who gonna have the next banger? Pop Smoke or Fivio Foreign</p>
               <br />
             </a>
+            <a
```

Comment:
make a card component and map through the data instead of having all this hardcoded